src/main/java/app/studnicki/ox/Round.java

```diff
@@ -0,0 +1,55 @@
+package app.studnicki.ox;
+
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+import java.util.HashMap;
+import java.util.Map;
+
+class Round {
+  final int dimension;
+  Map<Integer, Sign> board = new HashMap<>();
```

Comment:
why not private ?

Reply:
I use this in other when I want to print the board.